x/gov/types/proposal.go

```diff
@@ -0,0 +1,358 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Proposal defines a struct used by the governance module to allow for voting
+// on network changes.
+type Proposal struct {
+	Content `json:"content"` // Proposal content interface
+
+	ProposalID       uint64         `json:"proposal_id"`        //  ID of the proposal
```

Comment:
`id` is cleaner for clients
```suggestion
	ProposalID       uint64         `json:"id"`        //  ID of the proposal
```